fpdf/table.py

```diff
@@ -83,6 +88,14 @@ def __init__(
         self._width = fpdf.epw if width is None else width
         self._wrapmode = wrapmode
         self.rows = []
+
+        from fpdf.fpdf import get_padding_tuple  # Avoid circular import
```

Comment:
A cleaner solution to this problem would be to move `get_padding_tuple()` to another source file.

You can for example simply move it in there, in `fpdf/table.py`

Reply:
moved to table